compiler/rustc_llvm/llvm-wrapper/CoverageMappingWrapper.cpp

```diff
@@ -23,10 +23,17 @@ extern "C" void LLVMRustCoverageWriteFilenamesSectionToBuffer(
     const char* const Filenames[],
     size_t FilenamesLen,
     RustStringRef BufferOut) {
+#if LLVM_VERSION_GE(12,0)
```

Comment:
These checks should be for >= 13, not >= 12.

Reply:
Oh, so I was mislead by this version check that this was changed in LLVM 12, but not yet included in our submodule. So this wouldn't need a submodule update after all.

Reply:
Ah, my mistake. Yes, these should be 13, and if you're pinning to llvm 12 rather than their main this shouldn't require a matching submodule update.

(I'm still getting my bearings with LLVM version numbers.)